modules/boss/src/main/ts/ephox/boss/api/SpecialGene.ts

```diff
@@ -0,0 +1,4 @@
+import { Gene } from './Gene';
+
+export const SpecialGene = (id: string, children: Gene[]): Gene =>
+  Gene(id, 'SPECIAL_GENE', children, {}, {});
```

Comment:
Maybe we should export this string as a const and import it in `Properties.ts`?

Reply:
I just realised they're all explicitly compared as strings in there... so I guess keep it this way for consistency?

Reply:
It didn't take long, so I decided to both export the string as a const *and* have it be consistent with the rest (just by doing all of it)